test/integration/default/default_spec.rb

```diff
@@ -0,0 +1,406 @@
+# encoding: utf-8
+
+# Inspec test for CIS_2012r2_L1
+#
+# Copyright (c) 2017 Matt Tunny, All Rights Reserved.
+#
+# The Inspec reference, with examples and extensive documentation, can be
+# found at http://inspec.io/docs/reference/resources/
+
+# WinLogon Tests
+describe registry_key('HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\Winlogon') do
+  its('PasswordExpiryWarning') { should eq 14 }
+  its('ScreenSaverGracePeriod') { should eq '5' }
+  its('AllocateDASD') { should eq '0' }
+  its('ScRemoveOption') { should eq '1' }
+  its('CachedLogonsCount') { should eq '4' }
+end
+
+# LSA tests
+describe registry_key('HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Lsa') do
```

Comment:
I think we may want to use `controls` and add references to the specific cis benchmarks. Like 
```
control 'lsa-1' do
  impact 1.0

  title 'LSA Network access'
  desc 'Do not allow anonymous enumeration of SAM accounts and shares' to `Enabled`'

  tag cis: '2.3.11.3'
  ...
  tag remediation: 'https://github.com/dev-sec/chef-windows-hardening'

  ref 'CIS Windows 2012 R2', url: 'https://benchmarks.cisecurity.org/tools2/windows/CIS_Microsoft_Windows_Server_2012_R2_Benchmark_v1.1.0.pdf'

  describe registry_key()
     ...
  end
end
```

Further examples are available here: https://github.com/chef/inspec/blob/master/examples/profile/controls/meta.rb